src/permitInfo/calculatePermitInfo.ts

```diff
@@ -0,0 +1,154 @@
+import {getTokenPermitInfo, PermitInfo} from 'alfetopito-permit-utils' // import {getTokenPermitInfo, PermitInfo} from '@cowprotocol/permit-utils' // TODO: maybe make the args nicer?
+import * as path from 'node:path'
+import {ethers} from 'ethers'
+import {readFileSync, writeFileSync} from 'node:fs'
+import {JsonRpcProvider} from '@ethersproject/providers'
+
+// TODO: maybe make the args nicer?
+// Get args from cli: chainId, optional token lists path, optional rpcUrl
+const [, , chainId, tokenListPath, rpcUrl] = process.argv
+
+const BASE_PATH = path.join('src/public/')
+
+async function calculatePermitInfo(
+  chainId: number,
+  tokenListPath: string | undefined,
+  rpcUrl: string | undefined,
+): Promise<void> {
+  // Load existing permitInfo.json file for given chainId
+  const permitInfoPath = path.join(BASE_PATH, `PermitInfo.${chainId}.json`)
+
+  let allPermitInfo: Record<string, PermitInfo> = {}
+
+  // Load existing permitInfo.json file for given chainId if it exists
+  try {
+    allPermitInfo = JSON.parse(readFileSync(permitInfoPath, 'utf8')) as Record<string, PermitInfo>
+  } catch (e) {
+    console.info(`File ${permitInfoPath} not found. It'll be created.`, e)
```

Comment:
a bit verbose but fine. Just at least delete the error printing. But i think is nicer to just check if the file exists, 

Also, maybe this is handy here too.

https://stackoverflow.com/questions/21194934/how-to-create-a-directory-if-it-doesnt-exist-using-node-js